### PR TITLE
fix menu closing when clicking an unfocusable element in the menu

### PR DESCRIPTION
**Pull request by taylorcode**
I removed handling the tap event in Menu.js because the logic in `handleBlur` appears to handle all of the menu opening/closing cases. Was there a reason I'm unaware of for the logic in `handleTap` in Menu.js?

Also, should I run `prepublish` and commit the artifacts alongside these source changes?

### CODE REVIEW

**Comment by davidtheclark** [comment]
@taylorcode: Have you tested on a mobile phone whether the menu still closes if you tap outside it?

**Comment by taylorcode** [comment]
@davidtheclark good call, I will test that!

**Comment by taylorcode** [comment]
your suspicion was correct -- the menu did not close when clicking outside on mobile. I tested on mobile/desktop all the interactions I could think of. I updated the commit, let me know if I can improve anything

File: src/Menu.js [hunk 1]
```diff
@@ -39,16 +54,29 @@ module.exports = class extends React.Component {
...
-  };
+    if (this.document) {
+      this.tapListener = createTapListener(this.document, this.handleTap);
+    }
+  }
+
+  addTapStartListener = () => {
+    if (this.document) {
+      this.document.addEventListener('mousedown', this.handleTapStart);
+      this.document.addEventListener('touchstart', this.handleTapStart);
```
Comment (davidtheclark) [inline src/Menu.js]: Why use these event listener instead of the tap listener that was created at line 70?

**Comment by justo** [comment]
It would be great to see this fixed and merged. This bug applies to scrollbars inside the dropdowns too, so users can't use the scrollbar at all, the menu closes when they click it.

**Comment by davidtheclark** [comment]
@taylorcode: Any plans to finish off this PR?

**Comment by davidtheclark** [comment]
Closing due to lack of activity. Please re-open if you'd like to resume.

**Comment by justo** [comment]
I think I'm going to take over this PR as it looks like it's been abandoned.

**Comment by davidtheclark** [comment]
@justo 🍕

**Comment by justo** [comment]
My apologies, it looks like you actually fixed this in the latest version 👍

**Comment by davidtheclark** [comment]
Oh, I forgot 😆

**Pull request closed without merging**